src/config.rs

```diff
@@ -527,10 +527,13 @@ impl Config {
 
 fn find_global_config_file() -> Result<PathBuf> {
     dirs::home_dir()
-        .ok_or_else(|| format_err!("Could not find home dir"))
-        .map(|mut path| {
-            path.push(CONFIG_RC_FILE_NAME);
-            path
+        .map(|p| p.join(CONFIG_RC_FILE_NAME))
+        .filter(|p| p.exists())
+        .or(dirs::config_dir()
+            .map(|p| p.join("sentry/sentrycli.ini"))
```

Comment:
We should extract `"sentry/sentrycli.ini"` into a constant, like `CONFIG_RC_FILE_NAME`